### DISCUSSION TITLE
No stack is created by default

### DISCUSSION THREAD

**Discussion by qlibp** [LeonHeidelbach/trailblazer.nvim#46] [Q&A] [answered]
When I try using trailblazer and adding some trail points, no default stack is created for me, and when I call `openmarklist`, error thrown.

Then I try using `addtrailmarkstack` then calling `openmarklist`, it still throws error.

Finally, I get it to work by `newtrailmark`->`addtrailmarkstack`->`switchtrailmarkstack`->`openmarklist`.

I think the default behavior is not quite user-friendly...

**Comment by LeonHeidelbach** [comment]
> When I try using trailblazer and adding some trail points, no default stack is created for me, and when I call `openmarklist`, error thrown.
> 
> 
> 
> Then I try using `addtrailmarkstack` then calling `openmarklist`, it still throws error.
> 
> 
> 
> Finally, I get it to work by `newtrailmark`->`addtrailmarkstack`->`switchtrailmarkstack`->`openmarklist`.
> 
> 
> 
> I think the default behavior is not quite user-friendly...

Can you post your config? This should not happen. By default TrailBlazer should create the `default` stack even if you have not created any trail marks yet.

**Reply by qlibp** [reply]
I don't add any configuration to the plugin, just using the default one.

**Reply by LeonHeidelbach** [reply, marked as answer]
But you do call the setup function through your plugin manager right? Without calling the setup function the plugin will not work.

```lua
-- Using packer
use({
    "LeonHeidelbach/trailblazer.nvim",
    config = function()
        require("trailblazer").setup()
    end,
})
```

**Reply by qlibp** [reply]
Don’t forget to run setup()
